### PR TITLE
Updated for calls

### PR DESCRIPTION
**Pull request by theSociableme**

### CODE REVIEW

File: source/includes/_capture_external_event_api.md [hunk 1]
```diff
@@ -1,5 +1,179 @@
...
+Capture External Event API is a public ReachLocal external interface for vendors to use.
+This endpoint is used to inform us of confirmed leads for First Party Data
+
+### Parameters
+
+**Post Body Parameters**
+
+|Parameter | Required | Description|
+|---|---|---|---|---|
+|provider|yes|The provider of the First Party Day.  Example:  "Acme Co."|
```
Comment (gary-moon) [inline source/includes/_capture_external_event_api.md:25]: `Day` -> `Data`?

File: source/includes/_capture_external_event_api.md [hunk 2]
```diff
@@ -1,5 +1,179 @@
...
+
+**Call Object Parameters**
+
+|Data Field|Required|Datatype|Description|
+|---|---|---|---|
+|gmaid|yes|string|Global Master Advertiser ID|
+|phone|yes|string|Normalized phone number of the lead|
+|duration|yes|number|The duration of the call in seconds|
+|recording_url|yes|string|The URL address to an audio recording of the call.|
+|occured_at|no|string|The date and time that the call occurred|
```
Comment (gary-moon) [inline source/includes/_capture_external_event_api.md:36]: Shouldn't this be `occurred_at` (with two `r`s)?

File: source/includes/_capture_external_event_api.md [hunk 3]
```diff
@@ -1,5 +1,179 @@
...
+|tag|yes|integer|1: High Value, 2: Mid Value, 3: low Value|
```
Comment (gary-moon) [inline source/includes/_capture_external_event_api.md:37]: Should `low` be capitalized to stay consistent with `High` and `Mid`?

**Review by neelohitha — COMMENTED** [review]
this should be confirmed_calls

**Pull request closed without merging**